ng-dev/release/publish/actions/branch-off-next-branch.ts

```diff
@@ -134,4 +134,26 @@ export abstract class BranchOffNextBranchBaseAction extends ReleaseAction {
     info(green(`  ✓   Pull request for updating the "${nextBranch}" branch has been created.`));
     info(yellow(`      Please ask team members to review: ${nextUpdatePullRequest.url}.`));
   }
+
+  /** Prompt the user to determine the new version to be used in the "next" branch. */
+  private async _promptForNewNextVersion() {
+    const {branchName, version} = this.active.next;
+    const newMinorVersion = semverInc(version, 'preminor', 'next');
+    const newMajorVersion = semverInc(version, 'premajor', 'next');
+
+    info(
+      `Moving "${branchName}" into ${this.newPhaseName} requires the version in the "next" ` +
+        'branch to be updated',
+    );
+
+    return await prompt<semver.SemVer>({
+      name: 'newNextVersion',
+      message: 'Please select the new version for the next branch',
+      type: 'list',
+      choices: [
+        {name: `Minor (${newMinorVersion.format()})`, value: newMinorVersion},
+        {name: `Major (${newMajorVersion.format()})`, value: newMajorVersion},
```

Comment:
Personally I'm not 100% convinced of prompting when branching-off because switching to major-mode is rather uncommon, compared to switching to minor (which is common), which is why I think it should be only done separate action (obviously it still exists; but I think it should be the only option).

This would match up with the goal of making it really hard to mess things up (asking every time will require the caretaker to think about this and know beforehand what `next` will be. this is what we initially wanted to avoid).

If you feel strongly though, then I think the prompt should rather be a two-step process like:

```
Should the `next` branch target the upcoming new major? Yes
Are you sure? Yes
```